src/configs/envConfig.js

```diff
@@ -76,12 +76,12 @@ const envVars = {
   },
   staging: {
     ENVIRONMENT: STAGING,
-    TX_DETAILS_URL_ETHEREUM: 'https://kovan.etherscan.io/tx/',
-    TX_DETAILS_URL_BINANCE: 'https://bscscan.com/tx/',
+    TX_DETAILS_URL_ETHEREUM: 'https://goerli.etherscan.io/tx/',
+    TX_DETAILS_URL_BINANCE: 'https://testnet.bscscan.com/tx/',
     TX_DETAILS_URL_POLYGON: 'https://mumbai.polygonscan.com/tx/',
-    TX_DETAILS_URL_XDAI: 'https://blockscout.com/xdai/mainnet/tx/',
+    TX_DETAILS_URL_XDAI: 'https://blockscout.com/poa/sokol/tx/',
     TX_DETAILS_URL_AVALANCHE: 'https://testnet.snowtrace.io/tx/',
-    TX_DETAILS_URL_OPTIMISM: 'https://kovan-optimistic.etherscan.io/tx/',
+    TX_DETAILS_URL_OPTIMISM: 'https://blockscout.com/optimism/goerli/tx/',
     NETWORK_PROVIDER: 'kovan',
     COLLECTIBLES_NETWORK: 'rinkeby',
     OPEN_SEA_API: 'https://rinkeby-api.opensea.io/api/v1',
```

Comment:
These three are deprecated also, just wondering where these are used?

```suggestion
    NETWORK_PROVIDER: 'kovan',
    COLLECTIBLES_NETWORK: 'rinkeby',
```

Reply:
OPEN_SEA_API is used to get NFTs and NFTs transaction history.

Reply:
So I noticed that Kovan and Rinkeby are still defined as values here, but these networks are deprecated. Do you know where we're using NETWORK_PROVIDER and COLLECTIBLES_NETWORK?

Reply:
Usage for NETWORK_PROVIDER 
  1. check is production or staging status
  2. key wallet in find provider
  3. archanova in for environmentNetwork
  4. For network in user properties
 
 Usage for COLLECTIBLES_NETWORK
   1. In transferERC721 
   2. System information
   3. use in networkProvider for wallet instance connect
   
Should I need to update this?

Reply:
I think let's leave this for later